python/ray/worker.py

```diff
@@ -1736,7 +1736,12 @@ def method(self):
       number of times that the actor should be reconstructed when it dies
       unexpectedly. The minimum valid value is 0 (default), which indicates
       that the actor doesn't need to be reconstructed. And the maximum valid
-      value is ray.ray_constants.INFINITE_RECONSTRUCTIONS.
+      value is ray.ray_constants.INFINITE_RECONSTRUCTION.
+    * **max_retries**: Only for *remote functions*. This specifies the maximum
```

Comment:
I don't like the inconsistency between `max_reconstructions` for actors and `max_retries` for tasks. Thoughts about this? @pcmoritz

Reply:
Yeah I agree. Maybe `max_reconstructions` is a bit clearer? Don't have a strong preference though.